index.html

```diff
@@ -0,0 +1,416 @@
+            <a href="#">
+              <svg
+                viewBox="0 0 24 24"
+                preserveAspectRatio="xMidYMid meet"
+                focusable="false"
+                style="pointer-events: none; display: block"
+                class="style-scope yt-icon"
+              >
+                <g class="style-scope yt-icon">
+                  <path
+                    d="M11,7l6,3.5L11,14V7L11,7z M18,20H4V6H3v15h15V20z M21,18H6V3h15V18z M7,17h13V4H7V17z"
+                    class="style-scope yt-icon"
+                  ></path>
+                </g>
+              </svg>
+              Library
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <svg
+                viewBox="0 0 24 24"
+                preserveAspectRatio="xMidYMid meet"
+                focusable="false"
+                style="pointer-events: none; display: block"
+                class="style-scope yt-icon"
+              >
+                <g class="style-scope yt-icon">
+                  <path
+                    d="M14.97,16.95L10,13.87V7h2v5.76l4.03,2.49L14.97,16.95z M22,12c0,5.51-4.49,10-10,10S2,17.51,2,12h1c0,4.96,4.04,9,9,9 s9-4.04,9-9s-4.04-9-9-9C8.81,3,5.92,4.64,4.28,7.38C4.17,7.56,4.06,7.75,3.97,7.94C3.96,7.96,3.95,7.98,3.94,8H8v1H1.96V3h1v4.74 C3,7.65,3.03,7.57,3.07,7.49C3.18,7.27,3.3,7.07,3.42,6.86C5.22,3.86,8.51,2,12,2C17.51,2,22,6.49,22,12z"
+                    class="style-scope yt-icon"
+                  ></path>
+                </g>
+              </svg>
+              History
+            </a>
+          </li>
+        </ul>
+      </div>
+      <div class="contents">
+        <div class="tag-bar">
+          <a href="#" class="active">All</a>
+          <a href="#">Live</a>
+          <a href="#">Music</a>
+        </div>
+
+        <div class="videos">
+          <div class="main-video">
+            <iframe
+              width="560"
+              height="315"
+              src="https://www.youtube.com/embed/lJIrF4YjHfQ"
+              title="YouTube video player"
+              frameborder="0"
+              allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+              allowfullscreen
+            ></iframe>
+
+            <div class="video-info">
+              <h2>Embed videos and playlists</h2>
+              <div class="details">
+                <div class="meta">327,447 views • May 13, 2016</div>
+                <div class="interact">
+                  <a href="#">Like</a>
+                  <a href="#">Dislike</a>
+                  <a href="#">Share</a>
+                  <a href="#">Save</a>
+                </div>
+              </div>
+            </div>
+            <div class="channel-info">
+              <div class="channel">
+                <img
+                  src="https://yt3.ggpht.com/ccPr80rfkOgsE0TMP8S8vEfP85gl12XzUGtySPFFYNMhxlQ62W7ijksmUIXv6fCBC1jBmoEqaA=s48-c-k-c0x00ffffff-no-rj"
+                  alt=""
```

Comment:
Add relevant text in `alt` for images

Reply:
[OPTIONAL]: Please always your comments as suggestions instead of orders.
For example:
This is a polite suggestion: "I think it would be better to use semantic tags"